vita3k/renderer/src/transfer.cpp

```diff
@@ -130,78 +138,143 @@ COMMAND(handle_transfer_copy) {
     const uint32_t colorKeyMask = helper.pop<uint32_t>();
     SceGxmTransferColorKeyMode colorKeyMode = helper.pop<SceGxmTransferColorKeyMode>();
     const SceGxmTransferImage *images = helper.pop<SceGxmTransferImage *>();
-    const SceGxmTransferImage &src = images[0];
-    const SceGxmTransferImage &dst = images[1];
+    const SceGxmTransferFormat src_fmt = images[0].format;
+    const SceGxmTransferFormat dst_fmt = images[1].format;
     SceGxmTransferType src_type = helper.pop<SceGxmTransferType>();
     SceGxmTransferType dst_type = helper.pop<SceGxmTransferType>();
 
-    if (src.format != dst.format) {
-        LOG_ERROR_ONCE("Unhandled format conversion from {} to {}", log_hex(fmt::underlying(src.format)), log_hex(fmt::underlying(dst.format)));
+    if (src_fmt != dst_fmt) {
+        LOG_ERROR_ONCE("Unhandled format conversion from {} to {}", log_hex(fmt::underlying(src_fmt)), log_hex(fmt::underlying(dst_fmt)));
         delete[] images;
         return;
     }
 
-    if (colorKeyMask != SCE_GXM_TRANSFER_COLORKEY_NONE && src.format != SCE_GXM_TRANSFER_FORMAT_U8U8U8U8_ABGR) {
-        LOG_ERROR_ONCE("Transfer copy with non-zero key mask not handled for format {}", log_hex(fmt::underlying(src.format)));
+    if (colorKeyMask != SCE_GXM_TRANSFER_COLORKEY_NONE && src_fmt != SCE_GXM_TRANSFER_FORMAT_U8U8U8U8_ABGR) {
```

Comment:
```suggestion
    if (mode != SCE_GXM_TRANSFER_COLORKEY_NONE && src_fmt != SCE_GXM_TRANSFER_FORMAT_U8U8U8U8_ABGR) {
```
Not forget fix this fail ^^

Reply:
oh indeed!